src/Modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,5 +1,95 @@
-import React from "react";
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.png";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useToast } from "../../../../Context/ToastContext";
+import { FormData } from "../../../../interfaces/Auth";
 
 export default function ForgetPass() {
-  return <>ForgetPass</>;
+	const [spinner, setSpinner] = useState<boolean>(false);
+	const { showSuccessToast, showErrorToast } = useToast();
+
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm<FormData>();
+	const navigate = useNavigate();
+
+	const onSubmit = async (data: FormData) => {
+		setSpinner(true);
+
+		try {
+			const response = await axios.post(
+				`https://upskilling-egypt.com:3003/api/v1/Users/Reset/Request`,
+				data
+			);
+			showSuccessToast(
+				"Your request is being processed, please check your email"
+			);
+			navigate("/reset-pass");
+		} catch (error) {
+			if (axios.isAxiosError(error) && error.response) {
+				showErrorToast(error.response.data.message);
+			} else {
+				// Handle other types of errors here
+				showErrorToast("An error occurred.");
+			}
+		} finally {
+			setSpinner(false);
+		}
+	};
+
+	return (
+		<>
+			<div className="Auth-container vh-100 row align-items-center justify-content-center overflow-auto gx-0 flex-nowrap ">
```

Comment:
The indentation is too long, if we changed the prettier configuration, we need to restore it to default  as it will make our code easier to read.